finagle-core/src/main/scala/com/twitter/finagle/tracing/Id.scala

```diff
@@ -7,10 +7,10 @@ import java.lang.{Boolean => JBool}
 import scala.util.control.NonFatal
 
 /**
- * Defines trace identifiers.  Span IDs name a particular (unique)
- * span, while TraceIds contain a span ID as well as context (parentId
- * and traceId).
- */
+  * Defines trace identifiers.  Span IDs name a particular (unique)
```

Comment:
could you please revert this?

Reply:
Yeah man. I had copy/pasted the entirety of SpanId to overwrite a previous commit in which I attempted to extend all spans to 128-bit. My IDE settings auto-formatted to these changes on save, and I didn't catch it.  I also didn't catch the differing indent on new methods like `apply` below. I'll fix these up.

Do you all have IDE-specific formatting settings shared for IntelliJ?

Reply:
Ugh, what a pain :/

Reply:
Ah sounds like a pain :/

I'm not too familiar with intellij, but I did some poking around and I found that we have some kind of Twitter scheme for code style here and that I could export it as XML. This is a totally unofficial solution and I'm not gonna guarantee that it will actually format properly, but you're welcome to give it a shot: https://pastebin.com/ZL0UkQ98.